MBBSEmu.sln

```diff
@@ -7,13 +7,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MBBSEmu", "MBBSEmu\MBBSEmu.
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{D8C9ECC5-DE2F-4F08-8FF1-F39E04156DEE}"
 	ProjectSection(SolutionItems) = preProject
-		COMPATIBILITY.MD = COMPATIBILITY.MD
 		README.md = README.md
 	EndProjectSection
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MBBSEmu.Tests", "MBBSEmu.Tests\MBBSEmu.Tests.csproj", "{C43FA321-E88C-4769-8C4B-BA21FFE11DA3}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MBBSEmu.CPU.Benchmark", "MBBSEmu.CPU.Benchmark\MBBSEmu.CPU.Benchmark.csproj", "{C472176B-E917-4754-A896-1295E45D3628}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MBBSEmu.CPU.Benchmark", "MBBSEmu.CPU.Benchmark\MBBSEmu.CPU.Benchmark.csproj", "{C472176B-E917-4754-A896-1295E45D3628}"
```

Comment:
was this project GUID change intentional?

Reply:
Unintentional but not breaking. Not sure what caused it.

Reply:
Actually now the value is the same as MBBSEmu.Tests. You sure it doesn't break anything? :)

Reply:
It wasn't causing issues within Visual Studio, but went ahead and reverted it.